grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -926,7 +926,7 @@ class InventoryItemController {
                     log.error("Sorry, unable to add new item to shipment.  Please try again.")
                     flash.message = "${warehouse.message(code: 'inventoryItem.unableToAddItemToShipment.message')}"
                 } else {
-                    def productDescription = format.product(product: productInstance) + (inventoryItem?.lotNumber) ? " #" + inventoryItem?.lotNumber : ""
+                    def productDescription = format.product(product: productInstance).decodeHTML() + (inventoryItem?.lotNumber) ? " #" + inventoryItem?.lotNumber : ""
```

Comment:
My understanding of Grails taglibs is that they are primarily intended to be called via, e.g., `<format:product product=...>` from within a GSP page. In that context, any reserved HTML characters must be escaped.

In Grails 1 there's no way to disambiguate between calling a taglib from a GSP page vs from a controller. Controllers do _not_ want html escapes in their strings. So, when we call a taglib from a controller, we need to stick a `.decodeHTML()` at the end. Nits like this are scattered throughout this PR; I'm calling this one out because it sorts highest in the code review.